tox.ini

```diff
@@ -4,13 +4,13 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py3,docs
+envlist = py3, docs, benchmarks
 
 [travis]
 python =
-  3.6: py36, docs
-  3.7: py37
-  3.8: py38
+  3.6: py36, docs, benchmarks
+  3.7: py37, docs, benchmarks
```

Comment:
To not mix too many things regarding the ongoing discussion in #127, I would propose to stick to the previous behaviour to merge this PR, so adding `benchmarks` just to the `envlist` and 3.6, but not to 3.7 and 3.8. Depending on the conlcusion in #127 this could then be fixed alter in a different PR.

Reply:
Good point. Reverted back to the previous behavior.